### DISCUSSION TITLE
In the included example example_win32_directx12 - Menu -> Open does nothing

### DISCUSSION THREAD

**Discussion by Promootheus** [ocornut/imgui#6168] [New Users Build/Link/Run issues ONLY!]
Hi, in the included example example_win32_directx12 - Menu -> Open does nothing.
Am I wrong in expecting a file open dialog box to appear?
VS 2022

**Comment by ocornut** [comment]
> Am I wrong in expecting a file open dialog box to appear?

Yes, it does nothing:
```cpp
if (ImGui::MenuItem("Open", "Ctrl+O")) {}
```
This is to demonstrate the API.

I https://github.com/ocornut/imgui/commit/963839373cf376b29eb1a5b990cccd40b50d75f3#diff-0309a94fac8b8dac8c7f7477b14c3916f73de3ae27b830fccdb9d20d4357ffb3L3906 while removing the word "dummy" I made this section less clear. May need to add an extra comment.

**Reply by Promootheus** [reply]
Thanks, sorry to ask here. I can't find a Reddit or Discord group for general beginner advice on IMGui. I understand there is an FAQ, but is there a forum or group anywhere I could join?

**Reply by ocornut** [reply]
There's none AFAIK you can ask question here.
We had a Discord but I stopped here as I felt the nature of the medium made answers lower-quality and interesting problems less likely to be surfaced.
